src/main/java/org/wso2/lsp4intellij/client/languageserver/wrapper/LanguageServerWrapper.java

```diff
@@ -114,8 +114,7 @@
 import static org.wso2.lsp4intellij.requests.Timeout.getTimeout;
 import static org.wso2.lsp4intellij.requests.Timeouts.INIT;
 import static org.wso2.lsp4intellij.requests.Timeouts.SHUTDOWN;
-import static org.wso2.lsp4intellij.utils.ApplicationUtils.computableReadAction;
-import static org.wso2.lsp4intellij.utils.ApplicationUtils.invokeLater;
+import static org.wso2.lsp4intellij.utils.ApplicationUtils.*;
```

Comment:
please, do not use star-imports

Reply:
@nixel2007 It was done automatically by IntelliJ after importing `ApplicationUtils.pool`. Why should you not use star imports?

Reply:
https://rules.sonarsource.com/java/RSPEC-2208